src/Components/v2/CollectionsHubsNav.tsx

```diff
@@ -21,10 +22,10 @@ export const CollectionsHubsNav: FC<CollectionsHubsNavProps> = props => {
       ]}
       gridGap={20}
     >
-      {props.marketingCollections.map((hub, i) => (
+      {props.marketingHubCollections.slice(0, 6).map(hub => (
         <ImageLink
           to={`/collection/${hub.slug}`}
-          src={placeholderImages[i]}
+          src={resize(hub.thumbnail, { width: 357, height: 175 })}
```

Comment:
Is it aprpopriate that these images and the ones in the other (homepage nav) component scale their images to the same size?

(Could very well be, I just can't remember off the top)

Reply:
The collect page hubs could probably be a little smaller, but I thought this was close enough that it wasn't worth trying to identify the max size possible. I'll take a look though, now that you've prompted me to do so :)

Reply:
These are the right dimensions for the collect tiles, but the homepage tiles should actually be a little bigger - 375x184 at their max.

Unfortunately, the images I received from Jeffrey are 357x175. They don't look bad slightly upscaled right now, but when he gets back from vacation we might want to follow up to get slightly larger ones.